game/static/game/css/landing.css

```
@@ -333,15 +333,89 @@ body {
+    );
+    border: 1px solid rgba(255, 255, 255, 0.08);
+    border-radius: 32px;
+    padding: 3.5rem;
+    overflow: hidden;
+
     display: flex;
     flex-direction: column;
-    gap: 1.25rem;
+    gap: 1.8rem;
+
+    backdrop-filter: blur(14px);
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)